verification_app/src/config.js

```diff
@@ -11,6 +11,8 @@ const CERTIFICATE_SCAN_TIMEOUT = process.env.REACT_APP_CERTIFICATE_SCAN_TIMEOUT
 const CERTIFICATE_SIGNED_KEY_TYPE = process.env.REACT_APP_CERTIFICATE_SIGNED_KEY_TYPE || 'RSA';
 const certificatePublicKeyBase58 = process.env.REACT_APP_CERTIFICATE_PUBLIC_KEY_BASE58 || "DaipNW4xaH2bh1XGNNdqjnSYyru3hLnUgTBSfSvmZ2hi";
 
+const CERTIFICATE_STATUS_VC = process.env.REACT_CERTIFICATE_STATUS_VC || 'true';
```

Comment:
have you tested this?
also it should be REACT_APP_CERTIFICATE_STATUS_VC

Reply:
Haven't expected it, resolved